R/tt_as_df.R

```diff
@@ -238,11 +244,13 @@ as_result_df <- function(tt, spec = NULL,
         stat_name <- setNames(cell_stat_names[, col_i - min(only_col_indexes) + 1, drop = TRUE], NULL)
         stat <- setNames(ret_tmp[!col_label_rows, col_i, drop = TRUE], NULL)
         necessary_stat_lengths <- sapply(stat, length)
+        stat[sapply(stat, is.null)] <- NA
 
         # Truncating or adding NA if stat names has more or less elements than stats
         stat_name <- lapply(seq_along(stat_name), function(sn_i) {
           stat_name[[sn_i]][seq_len(necessary_stat_lengths[sn_i])]
         })
+        stat_name[sapply(stat_name, function(x) length(x) == 0)] <- NA
```

Comment:
Would this work? It's a bit cleaner :)

```suggestion
        stat_name[!nzchar(stat_name)] <- NA
```